darts/tests/models/components/test_LayerNormVariants.py

```diff
@@ -0,0 +1,24 @@
+from darts.logging import get_logger
+
+logger = get_logger(__name__)
+
+try:
+    import torch
+
+    TORCH_AVAILABLE = True
+except ImportError:
+    logger.warning("Torch not available. Loss tests will be skipped.")
+    TORCH_AVAILABLE = False
+
+
+if TORCH_AVAILABLE:
+    from darts.models.components.LayerNormVariants import RMSNorm, ScaleNorm
+    from darts.models.components.PowerNorm import MaskPowerNorm
+    from darts.tests.base_test_class import DartsBaseTestClass
+
+    class LayerNormVariantsTestCase(DartsBaseTestClass):
```

Comment:
These tests are good, but slightly minimal :) Could you maybe add a test checking correctness of the norm computation with some simple toy values?
Also it'd be good to add a test or two for the TFT, for instance checking that the outputs (on an untrained model) are different for different norm layers.